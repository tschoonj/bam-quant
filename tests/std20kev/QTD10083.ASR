File: C:\xrfdaten\2014KW30\chris\std20kev\std10083.spe
Start: 21.07.2014 18:38:08
Stop: 21.07.2014 18:39:48
BESSY: 299.0032039 298.9988154
IONI: 3.126601E-11 3.147349E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 79898
Run: 100000
Dead: 20
ICR: 64110
OCR: 0
Accumulated BESSY: 59699.9179285
Accumulated Ioni: 6.27224877E-09
N Accumulation: 200
Normfactor:  3.9908997e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Bi
exp_x 151.260406494141 151.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  83   2     10.828   2233453.      1409.     191.51
