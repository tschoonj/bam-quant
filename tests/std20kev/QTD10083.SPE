$DATA:
0 2000
18470
12080
5302
1567
369
66
13
8
3
1
2
4
1
0
0
4
148
1214
1423
1435
1268
1328
1349
1178
1199
1089
1192
1034
1039
1044
1002
967
1010
990
985
1003
885
897
899
845
787
790
787
740
702
637
608
644
630
625
587
576
594
561
572
569
517
587
477
541
492
532
524
501
492
520
531
490
482
489
487
476
493
440
454
475
503
444
437
443
458
453
468
488
450
447
470
460
438
440
509
511
544
539
559
605
517
513
541
497
444
483
462
502
478
463
441
521
486
499
556
548
553
610
700
967
1514
2429
3683
5098
6327
6731
7139
7023
6907
6576
5880
5040
3837
2715
1791
1121
857
653
693
652
704
601
595
519
535
412
435
459
468
458
486
534
544
485
429
416
399
391
411
412
422
465
473
517
508
474
466
479
390
384
361
349
364
343
308
292
284
322
349
307
332
316
338
363
357
324
321
342
328
373
311
326
312
317
315
327
335
345
299
303
310
345
330
340
318
324
288
314
293
311
275
302
284
319
281
275
288
310
271
289
308
316
292
267
303
286
293
296
307
322
301
290
285
323
302
299
348
287
350
295
276
279
304
283
288
297
276
265
290
277
274
264
281
282
278
320
300
325
312
292
283
291
317
296
289
322
325
363
372
432
490
526
488
494
415
411
351
347
304
295
304
252
266
265
290
297
309
288
303
314
311
300
298
306
325
308
318
312
300
354
294
301
314
287
326
283
293
261
288
328
280
298
302
288
318
310
332
372
334
337
367
398
378
340
354
354
279
309
294
287
300
329
291
322
284
322
312
292
258
270
304
300
289
346
284
275
322
332
314
306
317
295
297
328
311
365
284
311
322
295
261
326
280
325
300
331
344
310
281
273
286
344
322
298
322
337
335
285
346
347
315
364
358
320
323
309
344
326
315
334
328
322
326
323
347
336
326
305
309
341
345
373
354
361
324
331
321
336
349
342
364
332
351
357
361
330
377
353
398
404
338
372
331
386
357
361
412
357
361
367
365
397
387
371
371
391
424
402
383
388
406
407
444
445
435
494
476
495
540
553
611
629
701
753
698
756
759
721
753
801
896
1093
1453
2064
2805
3936
5556
7191
8639
9705
10454
10419
9766
8623
6968
5448
4120
2966
2168
1515
1163
992
902
827
778
832
762
759
709
732
668
687
662
626
663
717
652
676
732
679
748
746
710
782
810
841
873
905
958
1005
1073
1110
1053
1106
1114
1119
1179
1181
1235
1282
1281
1449
1614
1801
2102
2604
3389
4763
6921
10449
15475
23102
34018
48658
67764
91109
119475
148346
176411
197795
208141
207494
194766
170359
139291
107510
78033
53058
34231
21018
12574
7361
4411
2854
2012
1614
1428
1169
1124
1022
1071
956
898
952
871
864
808
762
737
690
748
738
718
777
840
1017
1343
1600
2050
2642
3218
3842
4189
4491
4558
4451
4161
3579
3064
2453
1906
1367
1217
1000
873
788
773
705
717
763
725
720
754
750
764
810
811
880
868
900
989
1063
1069
1189
1398
1710
2134
2447
2920
3497
4197
4689
4981
5331
5767
6251
7102
7888
9175
10355
11480
12486
12934
12752
12660
12396
12530
13961
16835
22430
31483
45250
64085
86479
111243
136421
159750
175953
183957
181650
171396
152303
129015
103904
81225
61369
45942
34779
26691
21080
17365
14459
12498
11015
9703
8966
8318
7744
7074
6457
5580
4635
3813
2984
2322
1628
1229
1006
823
666
673
627
650
692
640
669
709
704
659
577
503
454
390
334
301
248
206
188
173
152
152
134
155
141
156
131
138
153
166
164
146
163
176
169
158
151
167
158
160
180
187
190
163
202
192
238
267
317
322
409
511
556
656
764
764
855
825
799
743
739
603
552
504
519
516
570
696
943
1293
1874
2759
4205
5882
7667
10022
12389
14773
16284
17641
17767
17133
15546
13538
11284
9148
6774
5142
3876
2934
2457
2238
2282
2523
2858
3392
3822
4236
4836
5098
5253
5272
5123
4849
4289
3704
3119
2531
1886
1414
985
777
518
368
275
205
175
156
133
144
127
142
144
163
223
259
338
432
500
556
625
612
588
573
509
440
390
273
238
170
163
78
59
45
39
36
14
18
20
9
10
8
17
15
13
15
10
10
9
17
13
7
10
13
15
12
6
13
13
9
11
5
11
6
9
7
8
7
12
6
11
12
6
9
12
6
7
8
10
9
7
11
13
10
11
10
9
9
10
11
18
12
13
15
14
14
9
8
9
6
12
6
10
5
9
11
5
9
7
6
4
6
7
6
11
9
7
3
5
4
10
6
11
8
6
5
9
5
7
3
11
9
6
6
10
6
9
7
7
10
11
10
6
5
10
12
7
10
15
9
11
10
10
11
9
12
11
12
10
11
11
14
21
18
16
23
16
15
15
17
20
29
20
31
24
33
34
37
32
35
38
40
28
39
35
44
44
48
48
60
61
62
59
71
77
104
99
110
120
127
124
146
159
151
175
172
167
145
159
154
183
144
134
127
145
103
98
86
97
70
71
65
47
38
27
26
17
19
9
13
6
10
6
8
10
3
5
6
6
4
5
6
8
6
4
2
7
7
2
10
6
10
2
7
5
3
5
8
13
6
8
9
13
11
5
8
4
10
12
16
20
32
30
58
54
65
91
141
136
157
215
264
303
278
337
317
342
341
349
374
329
264
245
172
165
143
94
70
60
38
35
22
16
14
16
16
11
15
14
11
10
17
12
15
16
18
26
21
23
32
42
41
45
45
43
44
49
47
39
33
47
40
31
24
21
24
23
10
18
11
14
6
6
14
5
15
9
11
10
9
12
6
5
8
9
7
9
7
14
10
19
14
16
16
24
22
23
25
36
32
30
42
59
39
49
56
53
69
80
99
96
114
140
174
213
223
305
344
402
454
528
579
620
645
680
677
664
636
578
513
490
377
340
235
226
197
146
131
109
93
66
64
56
40
53
40
24
22
35
17
20
21
19
14
18
15
12
13
14
14
11
15
12
15
22
12
20
24
13
34
38
16
16
12
7
7
8
15
12
8
6
6
10
7
8
4
7
3
7
5
9
13
8
13
5
13
7
12
12
9
14
16
16
22
17
28
38
32
28
41
49
55
37
46
59
56
54
56
82
91
85
110
123
150
172
163
211
240
289
344
337
353
345
370
364
345
342
299
286
218
225
182
173
143
123
109
89
83
72
65
49
52
44
56
51
44
33
48
38
37
31
28
25
13
21
11
8
14
10
8
6
2
6
4
9
8
3
3
4
5
6
2
3
3
7
5
4
6
0
1
4
3
1
1
2
2
3
3
1
2
1
1
2
2
5
2
0
6
4
4
4
5
6
5
8
2
4
6
4
5
5
7
4
10
6
11
9
10
10
13
11
16
21
21
33
36
37
43
58
56
61
50
63
60
69
59
59
46
34
36
33
36
28
13
22
25
27
29
22
20
31
27
26
17
33
23
21
24
22
15
12
12
10
13
8
8
6
1
2
4
2
9
4
6
0
1
2
0
0
1
2
1
1
6
2
4
2
2
1
1
5
1
4
4
1
1
0
0
1
0
1
0
0
0
2
0
0
1
0
0
0
0
1
0
0
1
0
0
1
0
1
1
0
1
0
0
0
0
2
1
0
1
0
1
2
1
4
3
5
1
3
2
1
2
3
1
5
3
4
3
4
3
3
3
1
2
4
2
2
3
1
4
0
1
4
4
3
2
0
3
1
4
0
0
0
0
1
2
1
1
1
1
1
0
0
1
0
1
0
1
0
0
1
1
1
1
1
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
1
1
2
0
0
2
1
0
2
0
0
2
0
2
0
1
1
1
0
1
2
0
0
1
0
1
1
1
2
0
2
0
0
0
3
1
0
1
2
1
0
0
0
0
0
0
1
1
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
1
1
1
1
0
0
1
1
0
1
0
1
3
0
0
1
4
0
3
2
0
0
1
3
1
0
0
2
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
2
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
2
1
0
0
0
0
0
0
1
0
0
1
0
1
0
0
2
1
0
2
0
2
1
4
0
1
1
1
2
0
0
4
4
0
2
4
1
0
1
1
0
0
0
0
0
1
0
1
0
0
1
0
1
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
1
0
0
1
1
1
1
0
0
1
1
2
1
0
1
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10083.spe
Start: 21.07.2014 18:38:08
Stop: 21.07.2014 18:39:48
BESSY: 299.0032039 298.9988154
IONI: 3.126601E-11 3.147349E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 79898
Run: 100000
Dead: 20
ICR: 64110
OCR: 0
Accumulated BESSY: 59699.9179285
Accumulated Ioni: 6.27224877E-09
N Accumulation: 200
Normfactor:  3.9908997e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Bi
 
exp_x 151.260406494141 151.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  83   2     10.828   2233453.      1409.     191.51
