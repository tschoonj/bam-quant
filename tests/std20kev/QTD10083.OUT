     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:24:00  บ   
     บ  Spectrum: QTD10083.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        3 iterations done    บ   
     บ  ChiSquare =   188.9   last change =   .01%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -30.3 ๑   100.0   บ     -30.3 ๑      .3   บ   
     บ  GAIN (eV/ch)  บ     20.216 ๑   2.000   บ    20.216 ๑    .001   บ   
     บ det NOISE (eV) บ      114.4 ๑    40.0   บ     114.4 ๑     1.5   บ   
     บ  FANO factor   บ       .104 ๑    .050   บ      .104 ๑    .001   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Bi-L1                            290472. ๑    762.           บ   
     บ    L1M3       13.210    .37289      108314. ๑    285.    982.7  บ   
     บ              654.931    205.60         316.           9.86E-01  บ   
     บ    L1M2       12.691    .34774      101009. ๑    266.    520.2  บ   
     บ              629.258    202.80         337.           9.85E-01  บ   
     บ    L1N3       15.710    .10937       31768. ๑     85.    706.6  บ   
     บ              778.593    218.63         251.           9.88E-01  บ   
     บ    L1N2       15.582    .09174       26648. ๑     72.    809.3  บ   
     บ              772.262    217.98         256.           9.88E-01  บ   
     บ    L1O3       16.295    .02172        6309. ๑     22.    153.5  บ   
     บ              807.530    221.56         231.           9.87E-01  บ   
     บ    L1O2       16.271    .01882        5466. ๑     21.    153.5  บ   
     บ              806.343    221.44         232.           9.87E-01  บ   
     บ    L1M5       13.808    .01950        5663. ๑     23.     19.7  บ   
     บ              684.511    208.79         292.           9.86E-01  บ   
     บ    L1M4       13.700    .01302        3782. ๑     20.     49.4  บ   
     บ              679.169    208.22         296.           9.88E-01  บ   
     บ    L1N5       15.951    .00320         931. ๑     16.    243.1  บ   
     บ              790.514    219.84         243.           9.87E-01  บ   
     บ    L1N4       15.904    .00200         582. ๑     16.    349.9  บ   
     บ              788.189    219.60         244.           9.88E-01  บ   
     บ                                                                 บ   
     บ  2 Bi-L2                           1706548. ๑   1541.           บ   
     บ    L2M4       13.024    .77725     1326411. ๑   1198.   1270.4  บ   
     บ              645.730    204.60         323.           9.86E-01  บ   
     บ    L2N4       15.248    .16913      288627. ๑    261.   2107.2  บ   
     บ              755.740    216.28         268.           9.87E-01  บ   
     บ    L2M1       11.712    .02148       36660. ๑     38.    313.2  บ   
     บ              580.832    197.39         380.           9.85E-01  บ   
     บ    L2N1       14.773    .00560        9564. ๑     18.     29.2  บ   
     บ              732.244    213.84         257.           9.87E-01  บ   
     บ    L2O4       15.685    .02444       41702. ๑     41.    712.6  บ   
     บ              777.357    218.50         252.           9.88E-01  บ   
     บ    L2O1       15.551    .00120        2051. ๑     16.    934.9  บ   
     บ              770.728    217.82         257.           9.88E-01  บ   
     บ    L2M3       12.534    .00090        1533. ๑     19.    568.3  บ   
     บ              621.492    201.94         344.           9.84E-01  บ   
     บ                                                                 บ   
     บ  3 Bi-L3                           2907824. ๑   1834.           บ   
     บ    L3M5       10.839    .68965     2005392. ๑   1265.    191.5  บ   
     บ              537.649    192.44         420.           9.82E-01  บ   
     บ    L3M4       10.731    .07843      228061. ๑    145.    178.5  บ   
     บ              532.307    191.82         425.           9.81E-01  บ   
     บ    L3N5       12.980    .14081      409461. ๑    259.   1258.2  บ   
     บ              643.554    204.36         325.           9.86E-01  บ   
     บ    L3N4       12.955    .01576       45817. ๑     34.   1243.4  บ   
     บ              642.317    204.23         326.           9.86E-01  บ   
     บ    L3M1        9.420    .04081      118668. ๑     78.    163.1  บ   
     บ              467.459    184.12         438.           9.73E-01  บ   
     บ    L3N1       12.482    .01032       30012. ๑     27.    556.2  บ   
     บ              618.920    201.65         346.           9.84E-01  บ   
     บ    L3O1       13.259    .00221        6426. ๑     18.    477.2  บ   
     บ              657.355    205.87         314.           9.86E-01  บ   
     บ    L3O5       13.395    .02201       63988. ๑     44.    124.2  บ   
     บ              664.082    206.60         308.           9.87E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                           27438. ๑    168.           บ   
     บ               23.851    .29708        8341. ๑     52.      6.8  บ   
     บ             1181.293    256.49          64.           8.45E-01  บ   
     บ    L2N1       21.678    .16723        4695. ๑     30.      7.2  บ   
     บ             1073.800    246.95         102.           9.12E-01  บ   
     บ    L3M5       26.024    .13194        3704. ๑     24.      7.8  บ   
     บ             1288.785    265.68          40.           7.68E-01  บ   
     บ    L3N5       26.087    .04812        1351. ๑     10.      5.2  บ   
     บ             1291.891    265.94          39.           7.65E-01  บ   
     บ    L3M1       28.260    .04274        1200. ๑      9.      7.8  บ   
     บ             1399.384    274.82          19.           6.88E-01  บ   
     บ    L3N1       21.570    .03804        1068. ๑     12.      7.1  บ   
     บ             1068.458    246.47         104.           9.15E-01  บ   
     บ               23.743    .03379         949. ๑     10.     10.8  บ   
     บ             1175.950    256.02          66.           8.48E-01  บ   
     บ               20.259    .01979         556. ๑     12.     52.3  บ   
     บ             1003.609    240.52         134.           9.46E-01  บ   
     บ               24.052    .01914         537. ๑      8.      1.4  บ   
     บ             1191.217    257.35          61.           8.38E-01  บ   
     บ               22.432    .01758         494. ๑     10.      1.8  บ   
     บ             1111.102    250.30          87.           8.90E-01  บ   
     บ               26.225    .01700         477. ๑      7.      1.2  บ   
     บ             1298.710    266.52          38.           7.60E-01  บ   
     บ               23.530    .01685         473. ๑      9.     12.3  บ   
     บ             1165.409    255.10          69.           8.55E-01  บ   
     บ               25.703    .01497         420. ๑      7.     11.2  บ   
     บ             1272.902    264.34          44.           7.79E-01  บ   
     บ               26.535    .01225         344. ๑      6.      1.7  บ   
     บ             1314.042    267.80          34.           7.49E-01  บ   
     บ               28.708    .01088         306. ๑      4.      3.3  บ   
     บ             1421.535    276.62          16.           6.72E-01  บ   
     บ               24.234    .01067         300. ๑      8.      2.3  บ   
     บ             1200.232    258.13          58.           8.31E-01  บ   
     บ               26.407    .00948         266. ๑      6.      1.5  บ   
     บ             1307.725    267.27          36.           7.54E-01  บ   
     บ               22.551    .00611         172. ๑      9.      1.4  บ   
     บ             1116.983    250.83          85.           8.87E-01  บ   
     บ               25.979    .00547         154. ๑      6.      9.5  บ   
     บ             1286.549    265.50          40.           7.69E-01  บ   
     บ               24.724    .00543         152. ๑      7.      2.4  บ   
     บ             1224.475    260.22          52.           8.14E-01  บ   
     บ               23.324    .00526         148. ๑      9.      5.7  บ   
     บ             1155.196    254.21          72.           8.62E-01  บ   
     บ               26.419    .00479         134. ๑      6.      1.5  บ   
     บ             1308.302    267.32          35.           7.53E-01  บ   
     บ               25.497    .00467         131. ๑      7.      5.1  บ   
     บ             1262.688    263.48          46.           7.87E-01  บ   
     บ               28.592    .00425         119. ๑      4.      6.6  บ   
     บ             1415.795    276.15          17.           6.76E-01  บ   
     บ               30.496    .00346          97. ๑      3.      5.2  บ   
     บ             1509.982    283.67           7.           6.13E-01  บ   
     บ               24.668    .00285          80. ๑      7.      2.6  บ   
     บ             1221.700    259.98          52.           8.16E-01  บ   
     บ               28.461    .00275          77. ๑      4.      8.3  บ   
     บ             1409.308    275.63          18.           6.81E-01  บ   
     บ               27.939    .00242          68. ๑      5.       .5  บ   
     บ             1383.500    273.53          22.           6.99E-01  บ   
     บ               20.151    .00225          63. ๑     12.     88.3  บ   
     บ              998.267    240.02         136.           9.48E-01  บ   
     บ               23.944    .00218          61. ๑      8.      3.0  บ   
     บ             1185.875    256.89          63.           8.41E-01  บ   
     บ               21.462    .00216          61. ๑     10.      5.8  บ   
     บ             1063.115    245.98         106.           9.18E-01  บ   
     บ               27.123    .00196          55. ๑      5.      3.1  บ   
     บ             1343.130    270.21          28.           7.28E-01  บ   
     บ               23.422    .00192          54. ๑      8.      9.2  บ   
     บ             1160.067    254.63          70.           8.59E-01  บ   
     บ               30.944    .00176          49. ๑      2.      2.2  บ   
     บ             1532.133    285.41           5.           5.99E-01  บ   
     บ               29.296    .00175          49. ๑      4.      2.4  บ   
     บ             1450.622    278.96          13.           6.52E-01  บ   
     บ               25.612    .00159          45. ๑      7.      9.8  บ   
     บ             1268.395    263.96          45.           7.83E-01  บ   
     บ               28.643    .00154          43. ๑      4.      4.0  บ   
     บ             1418.323    276.36          17.           6.75E-01  บ   
     บ               27.785    .00142          40. ๑      5.      1.1  บ   
     บ             1375.888    272.91          23.           7.05E-01  บ   
     บ               26.427    .00139          39. ๑      6.      1.5  บ   
     บ             1308.700    267.35          35.           7.53E-01  บ   
     บ               24.126    .00121          34. ๑      8.      1.8  บ   
     บ             1194.890    257.67          60.           8.35E-01  บ   
     บ               22.633    .00113          32. ๑      9.      1.6  บ   
     บ             1121.026    251.18          84.           8.84E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
